analyses/tmb-compare-tcga/compare-tmb.Rmd

```diff
@@ -0,0 +1,273 @@
+---
+title: "Tumor Mutation Burden Comparison with TCGA data"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: C. Savonen for ALSF CCDL
+date: 2019
+---
+
+#### Purpose
+
+Plot the consensus tumor mutation burden statistics for PBTA in comparison to TCGA brain-related data. 
+
+#### Usage
+
+To run this from the command line, use:
+```
+Rscript -e "rmarkdown::render('analyses/tmb-compare-tcga/compare-tmb.Rmd', 
+                              clean = TRUE)"
+```
+
+_This assumes you are in the top directory of the repository._
+
+## Setup
+
+#### Packages and functions
+
+```{r}
+if (!("TCGAbiolinks" %in% installed.packages())) {
+  install.packages("TCGAbiolinks")
+}
+
+# magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+# Load in these functions so we can use `maf_to_granges`
+source(file.path("..", "snv-callers", "util", "wrangle_functions.R"))
+```
+
+Declare names of input and output directories.
+
+```{r}
+data_dir <- file.path("..", "..", "data")
+scratch_dir <- file.path("..", "..", "scratch")
+results_dir <- "results"
+plots_dir <- "plots"
+```
+
+Create output directories for this analysis. 
+
+```{r}
+if (!dir.exists(results_dir)) {
+  dir.create(results_dir)
+}
+if (!dir.exists(plots_dir)) {
+  dir.create(plots_dir)
+}
+```
+
+Custom function for plotting the TMB. 
+
+```{r}
+sina_plot <- function(data, colour = NULL, title) {
+  # Given a data.frame with `short_histology` and `tmb` information, make a sina plot from it
```

Comment:
I can imagine that what column we end up using will change, so to future proof this can you accept the column to use as an argument to this function?